File: 백준/Bronze/8370. Plane/Plane.c
#include<stdio.h>
int main(void){
    int n1,k1,n2,k2;
    int sum = 0;
    scanf("%d%d%d%d",&n1,&k1,&n2,&k2);
    sum = n1 * k1 + n2 * k2;
    printf("%d\n",sum);
    return 0;
}

File: 백준/Bronze/8370. Plane/Plane.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n1 = sc.nextInt(); // business class rows
        int k1 = sc.nextInt(); // business seats
        int n2 = sc.nextInt(); // economic class rows
        int k2 = sc.nextInt(); // economic seats
        
        int sum = n1 * k1 + n2 * k2;
        System.out.println(sum);
    }
}